### ISSUE TITLE
Some parts of Ui is not accessible for screen readers(blind and visually impaired)

### ISSUE THREAD

**Issue by hamidrezaabroshan** [rancher/rancher#35861]
I'm using the Nvda screen reader on windows to work with computers and recently joined a company as a developer.
When I wanted to use rancher, I found out that most of the buttons will not activate by pressing enter or space and Component's roles will be reported incorrectly by Screen reader.
E.G, on the first page, My screen reader says scope button but my colleagues say that it's a combobox.
How to reproduce and test:
Download and test free and open source Nvda screen reader and try to navigate the UI by arrow keys and tab and shift+tab.
Any component should be activated by pressing enter and roles should be reported by screen reader as same as cited user's perspective.
You can use <a href="https://www.w3.org/TR/WCAG20-TECHS/ARIA6.html"> Aria tag </a> to correct component's roles and search accessibility for screen reader in web design and also look at accessibility section for every framework that you are using for Ui.Also, I'm up for any help if possible.
Thanks.
